src/main/java/roomescape/theme/domain/DateRange.java

```diff
@@ -2,11 +2,15 @@
 
 import java.time.Clock;
 import java.time.LocalDate;
+import java.util.HashSet;
+import java.util.Set;
+import java.util.stream.Collectors;
 
 public class DateRange {
 
     private static final int DAYS_BEFORE_A_WEEK = 7;
     private static final int DAYS_BEFORE_YESTERDAY = 1;
+    public static final int TOW_MONTH = 2;
```

Comment:
```suggestion
    public static final int TWO_MONTH = 2;
```
개인적인 의견을 드리자면 매직넘버를 상수화하는 것은 좋지만,
이미 `plusMonths(2)` 에서 2개월을 더한다는 것을 충분히 유추할 수 있어서 상수화를 안해도 된다고 생각합니다.

Reply:
습관적으로 상수화를 진행했는데 이런 경우에는 굳이 상수화가 필요하지 않은 부분인 것 같습니다. 개선해 보도록 하겠습니다!